files/templates/additional-scrape-configs.template.yaml

```diff
@@ -52,8 +56,9 @@
   honor_timestamps: false
   scheme: https
   kubernetes_sd_configs:
-  - api_server: https://{{ .APIServerURL }}
-    role: node
+  - role: node
+{{ if not .IsInCluster }}
```

Comment:
You might also need to change the cluster_type label so I am wondering if we should not add templatize is as a variable

Reply:
Good catch, done.